src/app/login-form/login-form.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card class="loginForm">
```

Comment:
Hmm, the mat card styling looks a little bit weird, I'll put more details when I have more time.